sysbuild/Kconfig.secureboot

```diff
@@ -369,4 +369,41 @@ config SECURE_BOOT_GENERATE_DEFAULT_KMU_KEYFILE
 
 endif
 
+config SECURE_BOOT_MONOTONIC_COUNTER
+	bool "HW monotonic version counter"
+	depends on SECURE_BOOT_APPCORE
+	default y
+	help
+	  The counter stores the current firmware version in a way that ensures that the value
+	  can only increase. This is used to prevent malicious rollback of the firmware.
+	  An array of slots is used for the counter. One slot is used per
+	  counter update, and each new slot must have a larger value than the
+	  previous. Application versions are checked against the largest
+	  counter value before being booted.
+
+config SECURE_BOOT_NUM_VER_COUNTER_SLOTS
+	int "Number of monotonic counter slots used for the firmware version"
+	default 20 if BOOTLOADER_MCUBOOT
+	default 240
+	range 2 300 if SOC_NRF5340_CPUAPP || SOC_SERIES_NRF91X || SOC_SERIES_NRF54LX
+	range 2 1800 if SOC_SERIES_NRF52X
+	range 2 288 if SOC_SERIES_NRF51X
+	depends on SECURE_BOOT_MONOTONIC_COUNTER
+	help
+	  The number of monotonic counter slots available for the counter,
+	  i.e., the number of times the counter can be updated.
+	  The slots are 16 bits each. The number of slots must be an even
+	  number to ensure that the total size of header and slots is aligned on a 32-bit word.
```

Comment:
this doesn't seem true? You can set the number of slots to 7 if you want, sure on nrf52 it means you waste one but on nrf54l you waste nothing because each counter (in NVM) is 32 bits not 16 bits.